src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -997,7 +997,14 @@ void MFEMSidreDataCollection::LoadExternalData(const std::string& filename,
   else
   #endif
   {
-    grp->loadExternalData(path);
+    if(!group_name.empty())
+    {
+      grp->loadExternalData(path);
+    }
+    else
+    {
+      m_bp_grp->loadExternalData(path);
+    }
```

Comment:
@white238 and @nselliott -- This was necessary to get `loadExternalData()` working in our Windows configuration, which has MPI for axom and a serial mfem.

I am not sure if we have test coverage for the `if` part (where the users provide a `group_name`)